ChronosTankDrive/src/main/java/frc/robot/subsystems/Buttons.java

```diff
@@ -0,0 +1,9 @@
+package frc.robot.subsystems;
+
+public class Buttons {
```

Comment:
Why isn't this class included in Constants.java? It seems like it would be easier if all the constants were in one file.

Reply:
I thought the Constants.java file was just for the Shooter, which is why I put it in the shooter package. I think this makes it easier to organize. If someone wants to change a button, they don't have to search through a large constants class, they can just go to the buttons class.

Reply:
I think it's a question of file organization that the team should discuss. If Constants is only for the shooter, then it should be renamed to make that explicit.

Reply:
You could organize Constants.class with nested inner classes that represent logical groupings for the constants. 

```
public class Constants {
    public class Shooter{
        //constants for Shooter
    }
    public class Buttons {
        // constants for Buttons
    }
    // etc.
```

Reply:
I used the nested class option in my most recent commit.